lib/phases/cold-launch.js

```diff
@@ -34,29 +33,15 @@ var ColdLaunch = function(options) {
    * 5. Register to capture performance entries
    * 6. Start runs
    */
-
-  this.getDevice()
-    .then(function() {
-      return phase.device.log.clear();
-    })
-    .then(function() {
-      return phase.device.helpers.restartB2G();
-    })
-    .then(function() {
-      return phase.waitForB2GStart();
-    })
-    .then(function() {
-      return phase.getAppCoordinates();
-    })
-    .then(function() {
-      return phase.prime();
-    })
-    .then(function() {
-      return phase.capture();
-    })
-    .then(function() {
-      return phase.start();
-    });
+  this
+    .getDevice()
+    .then(() => this.device.clearLog())
```

Comment:
My fault, when I added the remote host stuff to the service, I forgot to say return in getDevice. I'll fix.